ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -420,19 +420,27 @@ private void printAsTable(KsqlEntity ksqlEntity) {
           )).collect(Collectors.toList());
     } else if (ksqlEntity instanceof Queries) {
       List<Queries.RunningQuery> runningQueries = ((Queries) ksqlEntity).getQueries();
-      columnHeaders = Arrays.asList("Query ID", "Kafka Topic", "Query String");
+      columnHeaders = Arrays.asList("Query ID", "Kafka Topic", "Query String", "Statistics");
       rowValues = runningQueries.stream()
           .map(runningQuery -> Arrays.asList(
               Long.toString(runningQuery.getId()),
               runningQuery.getKafkaTopic(),
-              runningQuery.getQueryString()
+              runningQuery.getQueryString(),
+              runningQuery.getStatistics()
           )).collect(Collectors.toList());
     } else if (ksqlEntity instanceof SourceDescription) {
-      List<SourceDescription.FieldSchemaInfo> fields = ((SourceDescription) ksqlEntity).getSchema();
+      SourceDescription ksqlEntity1 = (SourceDescription) ksqlEntity;
```

Comment:
Should we name this variable `sourceDescription`?

Reply:
yep